qa/acceptance/spec/centos/lib/install_spec.rb

```diff
@@ -0,0 +1,31 @@
+require_relative '../spec_helper'
+
+describe "artifacts", :platform => :centos do
+
+  shared_examples "installable" do |host, name|
+
+    before(:each) do
+      install("logstash-2.3.1-1.noarch.rpm", host)
+    end
+
+    it "is installed on #{name}" do
+      expect("logstash").to be_installed.on(host)
+    end
+
+    it "is running in #{name}" do
+      start_service("logstash", host)
+      expect("logstash").to be_running.on(host)
+      stop_service("logstash", host)
+    end
+
+    it "is removable on #{name}" do
+      uninstall("logstash", host)
+      expect("logstash").to be_removed.on(host)
+    end
+  end
+
+  config = ServiceTester.configuration
+  config.servers.each do |host|
+    it_behaves_like "installable", host, config.lookup[host]
+  end
+end
```

Comment:
sure is not necessary, but also not nice to pass this var thought a let statement, I guess the best way here might be pass with metadata, but not sure this is accessible in helpers, will need to investigate.